band_tracker/db/models.py

```diff
@@ -123,6 +123,32 @@ class GenreDB(Base):
     name: Mapped[str] = mapped_column(String, nullable=False)
 
 
+class MessageDB(Base):
+    __tablename__ = "message"
+
+    id: Mapped[UUID] = mapped_column(
+        UUID_PG(as_uuid=True),
+        primary_key=True,
+        server_default=alchemy_text("gen_random_uuid()"),
+    )
+    user_id: Mapped[UUID] = mapped_column(
+        UUID_PG(as_uuid=True),
+        ForeignKey("user.id", ondelete="CASCADE"),
+        nullable=False,
+        index=True,
+    )
+    tg_message_id: Mapped[int] = mapped_column(Integer, nullable=False, unique=True)
+    message_type: Mapped[MessageType] = mapped_column(
+        EnumDB(MessageType), nullable=False, index=True
+    )
+    timestamp: Mapped[datetime] = mapped_column(
+        DateTime, nullable=False, default=datetime.now()
+    )
+    active: Mapped[bool] = mapped_column(
+        Boolean, nullable=False, default=True, index=True
+    )
+
+
```

Comment:
add a new field, smt like "auto delete" or some shit like that. Cause now we are deleting all active interfaces excluding notifications, we want to have a name for this group. in the future we might not want to delete some certain active messages (idk we didnt test it like properly, anything can change) or maybe some new interfaces will appear.